bin/pycbc_banksim

```diff
@@ -374,7 +374,7 @@ with ctx:
                                       template_sample_rate,
                                       filter_N, options.filter_sample_rate)
 
-            h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
+                h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
```

Comment:
@spxiwh on a last pass, I noticed this change. I think the "missing" indentation is actually needed to enable the `--filter-high-freq-cutoff-isco-factor` option. Could you double check before I merge?